packages/redux-query/src/lib/update.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import type { Update, Entities, OptimisticUpdate, Rollback } from '../types';
+
+export const updateEntities = (
+  update: Update = {},
+  entities: Entities = {},
+  transformed: Entities = {},
+): Entities => {
+  // If update is not supplied, then no change to entities will be made
+
+  return Object.keys(update).reduce((accum, key: string) => {
+    accum[key] = update[key](entities[key], transformed[key]);
+
+    return accum;
+  }, {});
+};
+
+export const optimisticUpdateEntities = (
+  optimisticUpdate: OptimisticUpdate,
```

Comment:
I think it's better to have these ones written the same way as the ones here: https://github.com/amplitude/redux-query/pull/129/files#diff-fe52475398e3e004625a25aaf3263de2R6 (with defaults)